### ISSUE TITLE
get_next_pass not working

### ISSUE THREAD

**Issue by tydlwav** [tydlwav/GSW-Sat-Tracking#2]
Needs to look into the Observer object in tracker.py

**Comment by purvigoel** [comment]
Was this fixed in the last push?

**Comment by tydlwav** [comment]
nope, I didn't have time for this. The main goal of the last push is to write the [user guide](jp.michaelmao.me/api), which has everything that's working in it.
